CampaignBase/media-event-provider-api.js

```diff
@@ -1,25 +1,52 @@
+function selectAppsflyerLeanSource() {
+  const ui = SpreadsheetApp.getUi();
+
+  const result = ui.alert(
+    "Dados Lean",
```

Comment:
```suggestion
    "Dados consolidados",
```

Consolidados fariam mais sentido pro usuário final.